plugins/by-name/neoconf/default.nix

```
@@ -0,0 +1,92 @@
+{ lib, ... }:
+let
+  inherit (lib.nixvim) defaultNullOpts;
+in
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
+  name = "neoconf";
+  originalName = "neoconf.nvim";
+  package = "neoconf-nvim";
+
+  maintainers = [ lib.maintainers.BoneyPatel ];
+
+  settingsOptions = {
+    local_settings = defaultNullOpts.mkStr ".neoconf.json" ''
+      Name of the local settings file for the plugin.
+    '';
+
+    global_settings = defaultNullOpts.mkStr "neoconf.json" ''
+      Name of the global settings file in your Neovim config directory.
+    '';
+
+    import = {
+      vscode = defaultNullOpts.mkBool true ''
+        Whether to import settings from local .vscode/settings.json files.
```

I think this makes sense to tag filepaths as code?

```suggestion
        Whether to import settings from local `.vscode/settings.json` files.
```

---

Updated with `.vscode/settings.json`.